src/mappings/spacewalk.ts

```diff
@@ -0,0 +1,184 @@
+import { EventHandlerContext } from '../processor'
+import { RedeemRequest, IssueRequest } from '../model'
+import { hexToSs58 } from './nabla/addresses'
+import { getOrCreateVault } from '../entities/vault'
+import { parseVaultId } from '../utils/vault'
+import { getVersionedStorage } from '../types/eventsAndStorageSelector'
+import { network } from '../config'
+import { IssueRequestStatus } from '../model/generated/_issueRequestStatus'
+import { RedeemRequestStatus } from '../model/generated/_redeemRequestStatus'
+
+export async function handleIssueRequest(ctx: EventHandlerContext) {
+    const { args } = ctx.event
+    const {
```

Comment:
Is the `asset` argument not interesting to us (or is it that we can derive it from the vault id)?

Reply:
We get it from the `vaultId` yes